ensime-mode.el

```diff
@@ -505,9 +506,10 @@ A prefix argument will add the report to the kill ring."
                          (insert-file-contents filePath)
                          (buffer-string))))
                  (progn "Not provided")))))
-    (when arg
+    (unless arg
       (kill-new info))
-    (message info)))
+    (message info)
```

Comment:
maybe we want to put this in a buffer of its own? Like `*ensime-troubleshooting*`. In fact, if we do that there is no reason to add it to the kill ring.